### PR TITLE
Dependency Fix for Conda Env

### PR DESCRIPTION
**Pull request by shrutij01**
## Description

add pytest in env.yml to be installed via conda and remove the example/
dir as it is no longer needed

## How I Tested

by running all the tests with pytest

- [x] All new code is formatted according to our style guide (for C++ run clang-format, for Python, run flake8 and fix all warnings).
- [x] All new functions/classes are documented and existing documentation is updated according to changes.
- [x] No commented code from testing/debugging is kept (unless there is a good reason to keep it).

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ dependencies:
 - python=3.8.5
 - numpy==1.19.1
 - pinocchio==2.4.7
+- pytest==6.2.3
```
Comment (luator) [inline environment.yml:8]: Can you test if it also works with pytest version 4.6.9?  Then it would match the version used in the Singularity image.

Comment (shrutij01) [reply environment.yml:8]: I get an error that the package pytest==4.6.9 is not found on the conda channel..

Comment (luator) [reply environment.yml:8]: Oh, okay.  Then let's keep v6 (it's anyway only for testing, so shouldn't be a problem).

**Review by luator — APPROVED** [review]

File: environment.yml (hunk 1 above)
Comment (shrutij01) [reply environment.yml:8]: okay

**Pull request merged**